examples/gradle-example/run.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -o errexit
 
-./gradlew run
+gradle run
```

Comment:
Unable to run example using `run.sh`

```sh
➜ ./run.sh
> Task :compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Cannot resolve external dependency com.autonomic.tmc:tmc-auth:2.0.3-alpha because no repositories are defined.
     Required by:
         project :
   > Cannot resolve external dependency io.grpc:grpc-stub:1.25.0 because no repositories are defined.
     Required by:
         project :
   > Cannot resolve external dependency io.grpc:grpc-netty:1.25.0 because no repositories are defined.
     Required by:
         project :
   > Cannot resolve external dependency io.netty:netty-tcnative-boringssl-static:2.0.28.Final because no repositories are defined.
     Required by:
         project :
   > Cannot resolve external dependency org.springframework.boot:spring-boot-starter:2.2.1.RELEASE because no repositories are defined.
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 420ms
1 actionable task: 1 executed
```

Reply:
Good catch. I think we still want to include the `oss` public repo in the `build.gradle` just without any credentials required to access the libs.

```
repositories {	
    maven {	
        url "https://autonomic.bintray.com/au-tmc-oss"
    }
}
```